src/phoenix/experimental/evals/utils.py

```diff
@@ -0,0 +1,27 @@
+"""
+Utility functions for evaluations.
+"""
+
+import json
+from urllib.error import HTTPError
+from urllib.request import urlopen
+
+import pandas as pd
+
+
+def download_benchmark_dataset(task: str, dataset_name: str) -> pd.DataFrame:
+    """Downloads an Arize evals benchmark dataset as a pandas dataframe.
+
+    Args:
+        task (str): Task to be performed.
+        dataset_name (str): Name of the dataset.
+
+    Returns:
+        pandas.DataFrame: A pandas dataframe containing the data.
+    """
+    url = f"http://storage.googleapis.com/arize-assets/phoenix/evals/{task}/{dataset_name}.jsonl"
+    try:
+        with urlopen(url) as f:
+            return pd.DataFrame([json.loads(line.decode()) for line in f.readlines()])
```

Comment:
what does `decode` do?
```suggestion
            return pd.DataFrame(map(json.loads, f.readlines()))
```

Reply:
also, these files are so large it may worth considering storing them as zipped files

Reply:
Good catch, `decode` was unnecessary here.

Reply:
addressed in [9eadbdc](https://github.com/Arize-ai/phoenix/pull/1241/commits/9eadbdce17df172747c37792a513e10a41908ee4)